packages/features/sales-escrows/modals/ConfirmSalesActionModal.tsx

```diff
@@ -28,6 +29,7 @@ interface ConfirmSalesActionModalProps {
   escrow?: EscrowRecord;
   offer?: EscrowRecord;
   onSaleCancelled?: () => void;
+  setInProcess?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
This should be a simple function signature like handleClose above. Forcing it as a state action forces how the function is implemented in the consuming component. Instead we want components to be loosely coupled.